chia/rpc/wallet_rpc_api.py

```diff
@@ -999,6 +1001,17 @@ async def send_notification(self, request) -> EndpointResult:
         await self.service.wallet_state_manager.add_pending_transaction(tx)
         return {"tx": tx.to_json_dict_convenience(self.service.config)}
 
+    async def sign_message(self, request) -> EndpointResult:
+        """
+        Given a derived P2 address, sign the message by its private key.
+        :param request:
+        :return:
+        """
+        puzzle_hash: bytes32 = decode_puzzle_hash(request["address"])
+        message = hexstr_to_bytes(request["message"])
```

Comment:
If I were calling this RPC and didn't read the docs, I would assume that `"message"` would be a regular string, and not a hex string.

I'd recommend renaming `"message"` to `"message_data"` to at least give a hint that a normal string isn't expected here. Or choose a name that self-documents its type like `"message_hex_string"` or something.